packages/labs/scoped-registry-mixin/src/test/scoped-registry-mixin_test.ts

```diff
@@ -108,4 +116,20 @@ suite('scoped-registry-mixin', () => {
     assert.isFalse($simpleGreeting instanceof SimpleGreeting);
     assert.isTrue($simpleGreeting instanceof HTMLElement);
   });
+
+  test(`simple-greeting should adopt the styles`, async () => {
```

Comment:
nit: I missed this in the previous review, but let's not put the test element name in the test name.

```suggestion
  test(`host element should apply static styles`, async () => {
```

Reply:
done